packages/internal/bridge/sdk/src/constants/bridges.ts

```diff
@@ -64,9 +64,18 @@ export const ETH_MAINNET_TO_ZKEVM_MAINNET: BridgeInstance = {
  * @typedef {Object} axelarAPIEndpoints - API endpoints for the testnet & mainnet Axelar environment configurations
  */
 export const axelarAPIEndpoints:Record<string, string> = {
-  testnet: 'https://testnet.api.gmp.axelarscan.io',
   mainnet: 'https://api.gmp.axelarscan.io',
-  devnet: '',
+  testnet: 'https://testnet.api.gmp.axelarscan.io',
+  devnet: 'https://testnet.api.gmp.axelarscan.io',
```

Comment:
I think the devnet endpoint is: https://stagenet.axelarscan.io/resources/chains

Reply:
updated.